v/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM taojy123/vlang:v0.1.24
```

Comment:
we can exceptionally use `:latest` @exastencil

Reply:
or use https://hub.docker.com/layers/gladmo/vlang/0.1.27-alpine/images/sha256-db6ca5e8ed213c1639d3cb42a0979b707f68c55fde926a0e869168cb6368a524?context=explore